README.md

```diff
@@ -81,9 +81,17 @@ drupal self-update
 You can ask for support at Drupal Console gitter chat room [http://bit.ly/console-support](http://bit.ly/console-support).
 
 ## Getting The Project To Contribute
-
 For information about how to run this project for development follow instructions at [setup instructions](https://gist.github.com/jmolivas/97bbd07f328217be3564a434c5bd2618).
 
+An optional way you can execute this:
+```
+composer require --dev drupal/console:dev-master
+```
+You might have to execute too:
+```
+composer require symfony/dom-crawler:2.8 symfony:debug:2.8
```

Comment:
@jmolivas  I think drupal/core require the version 2.7 of both them, if you try to install with composer the dev-master of drupal/console you need at least the version 2.8 so because of that composer says "Hey there is conflict of dependencies with 2 plugins do something"